.github/actions/colab/colab-python-py37-tf2/action.yml

```diff
@@ -1,2 +1,60 @@
-name: 'Run Colab Python Py37 tf2'
-description: 'Run Colab Python Py37 tf2'
\ No newline at end of file
+name: 'Run Colab Python Py37 tf1'
+description: 'Run Colab Python Py37 tf1'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Python 
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7.10'
+
+    - name: Install dependencies
+      shell: bash
+      run: |
+        apt-get update
+        apt-get install -y zip
+        python -m pip install --upgrade pip
+        python -m pip install --upgrade setuptools==58.0.4
+
+    - name: Setup env
+      shell: bash
+      run: |
+        if [ ! -d "/opt/conda/envs/py37" ]
+        then
+          conda create -n py37 -y python==3.7.10 setuptools==58.0.4
+        fi
+        source activate py37
+        pip install wheel
+        pip uninstall -y bigdl-friesian bigdl-friesian-spark3 bigdl-dllib bigdl-dllib-spark3 bigdl-orca pyspark bigdl-orca-spark3 bigdl-chronos bigdl-chronos-spark3 bigdl-friesian bigdl-friesian-spark3
+        pip uninstall -r .github/actions/python-requirements/requirements-colab-python-tf1-spark3.txt -y
+        pip install -i https://pypi.tuna.tsinghua.edu.cn/simple -r .github/actions/python-requirements/requirements-colab-python-tf1-spark3.txt -y
+
+      env:
+        BIGDL_ROOT: ${{ github.workspace }}
+        ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+    
+    - name: Run test
+      shell: bash
+      run: |
+        set -x
+        export CLASSPATH=.:${JAVA_HOME}/lib:${JAVA_HOME}/jre/lib:${JAVA_HOME}/lib/tools.jar:${JAVA_HOME}/lib/dt.jar
+        export PATH=${JAVA_HOME}/bin/:${JAVA_HOME}/jre/bin:${PATH}
+        export PATH=/opt/hostedtoolcache/maven/3.6.3/x64:$PATH 
+        source activate py37
+        chmod a+x $ANALYTICS_ZOO_HOME/python/orca/dev/colab-notebook/run-notebook-tf2.sh
+        chmod a+x python/orca/dev/colab-notebook/ipynb2py.sh
+        pip install --no-cache-dir tensorflow
+        pip uninstall -y bigdl bigdl-dllib bigdl-orca pyspark
+        pip install --pre --upgrade bigdl-orca
+        sh python/orca/dev/colab-notebook/run-notebook-ray.sh
+        sh python/orca/dev/colab-notebook/run-notebook-tf2.sh
+        pip uninstall -y bigdl bigdl-dllib bigdl-orca
+        source deactivate
+
+      env:
+        BIGDL_ROOT: ${{ github.workspace }}
+        ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+        FTP_URI: ftp://zoo:1234qwer@10.112.231.51
+        no_proxy: 10.112.231.51
```

Comment:
Can you put these envs as runner envs?

Reply:
Yes, when I pass the test, I will put this as runner envs.